dataPipelines/gc_thumbnails/cli.py

```diff
@@ -0,0 +1,40 @@
+import click
+
+from .utils import ThumbnailsCreator
+import typing as t
+
+####
+# CLI
+####
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.command(name='process')
+@click.option(
+    '-f',
+    '--file-name',
+    help="Name of the file you're extracting the thumbnail from",
+    type=str,
```

Comment:
type here should be `click.Path(dir_okay=False, exists=True, resolve_path=True)`, to resolve full path to a file that actually exists. It will still be `str` type inside the function - but click will make sure it's a valid path